packages/pn-personafisica-webapp/src/utility/events.ts

```diff
@@ -1,203 +1,291 @@
-  },
-  [TrackEventType.DELEGATION_DELEGATOR_REJECT]: {
-    category: 'delegation',
-    action: 'click/tap on reject action'
-  },
-  [TrackEventType.DIGITAL_DOMICILE_BANNER_CLOSE]: {
-    category: 'notification',
-    action: 'close the banner in notifications detail/list'
-  },
-  [TrackEventType.DIGITAL_DOMICILE_LINK]: {
-    category: 'notification',
-    action: 'click/tap on  the banner link in notifications detail/list'
-  },
-  [TrackEventType.NOTIFICATION_DETAIL_PAYMENT_ASSISTANCE]: {
-    category: 'notification',
-    action: 'contact assistance'
-  },
-  [TrackEventType.NOTIFICATION_DETAIL_PAYMENT_ERROR]: {
-    category: 'notification',
-    action: 'failed payment'
-  },
-  [TrackEventType.NOTIFICATION_DETAIL_PAYMENT_F24_FILE]: {
-    category: 'notification',
-    action: 'download the F24 file'
-  },
-  [TrackEventType.NOTIFICATION_DETAIL_PAYMENT_INTERACTION]: {
-    category: 'notification',
-    action: 'click on the pay button'
-  },
-  [TrackEventType.NOTIFICATION_DETAIL_PAYMENT_PAGOPA_FILE]: {
-    category: 'notification',
-    action: 'download the PagoPA file'
-  },
-  [TrackEventType.NOTIFICATION_DETAIL_PAYMENT_RELOAD]: {
-    category: 'notification',
-    action: 'reload the payment page'
-  },
-  [TrackEventType.NOTIFICATION_DETAIL_SINGLE_ATTACHMENT]: {
-    category: 'notification',
-    action: 'click/tap on  the banner link in notifications detail/list'
-  },
-  [TrackEventType.NOTIFICATION_FILTER_SEARCH]: {
-    category: 'notification',
-    action: 'click/tap on  the filter button in notifications filter'
-  },
-  [TrackEventType.NOTIFICATION_TABLE_PAGINATION]: {
-    category: 'notification',
-    action: 'change page',
-  },
-  [TrackEventType.NOTIFICATION_TABLE_ROW_INTERACTION]: {
-    category: 'notification',
-    action: 'go to notification detail'
-  },
-  [TrackEventType.NOTIFICATION_TABLE_SORT]: {
-    category: 'notification',
-    action: 'change notification sorting order',
-    getAttributes:  (payload: { [key: string]: string }) => ({
-      orderBy: payload.orderBy,
-      order: payload.order,
-    })
-  },
-  [TrackEventType.NOTIFICATION_TIMELINE_SINGLE_ATTACHMENT]: {
-    category: 'notification',
-    action: 'click/tap on  the banner link in notifications timeline'
-  },
-  [TrackEventType.NOTIFICATION_TIMELINE_VIEW_MORE]: {
-    category: 'notification',
-    action: 'click/tap on timeline view more'
-  },
-  [TrackEventType.USER_LOGOUT]: {
-    category: 'user',
-    action: 'user logs out from action'
-  },
-  [TrackEventType.USER_NAV_ITEM]: {
-    category: 'user',
-    action: 'click on a sidebar link'
-  },
-  [TrackEventType.USER_PRODUCT_SWITCH]: {
-    category: 'user',
-    action: 'switch product'
-  },
-  [TrackEventType.USER_VIEW_PROFILE]: {
-    category: 'user',
-    action: 'go to user profile'
-  },
-  [TrackEventType.USER_VIEW_CONTACTS_PROFILE]: {
-    category: 'user',
-    action: 'go to contacts from profile'
+  [TrackEventType.SEND_VIEW_PROFILE]: {
+    category: 'UX',
```

Comment:
@NicolaGiornetta aggiungiamo un enum EventCategory per la category UX e TECH o in una union di literals  EventCategory = "UX" | "TECH"